scripts/dev.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Here's what's happening: Makefile (a build tool) calls this script (a build tool) which calls poetry (another build tool). This layering feels a bit complex. Do you think there's an opportunity to simplify the flow?

Or could you help me understand why we have this?